cmd/dep/testdata/harness_tests/remove/specific/case2/testcase.json

```diff
@@ -2,6 +2,7 @@
   "commands": [
     ["remove", "github.com/not/used"]
   ],
+  "error-expected": "",
```

Comment:
I've noticed it elsewhere, as well. I assume the build doesn't break because we're not actually validating that file - it's just the testcase procedure. What actually strikes me as odd is that passing the `-update` flag writes these files out at all. That seems like the actual mistake bug.